model/channel_member_test.go

```diff
@@ -29,11 +29,10 @@ func TestChannelMemberIsValid(t *testing.T) {
 	o.NotifyProps = GetDefaultChannelNotifyProps()
 	o.UserId = NewId()
 	/*o.Roles = "missing"
-	o.NotifyProps = GetDefaultChannelNotifyProps()
-	o.UserId = NewId()
-	if err := o.IsValid(); err == nil {
-		t.Fatal("should be invalid")
-	}*/
+		o.NotifyProps = GetDefaultChannelNotifyProps()
```

Comment:
We should remove this comments part right? I mean, we have it in the version control system so no need to have it in the code

Reply:
I was actually playing conservative here, in case those comments were there for something, but I'm happy just removing them

Reply:
I tend to remove all the comments I found in the code because if needed you can get it from the version control system :)